pkgs/applications/networking/instant-messengers/scudcloud/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, python3Packages }:
+
+let version = "1.13"; in
+python3Packages.buildPythonPackage {
+  name = "scudcloud-${version}";
+  namePrefix = "";
+
+  src = fetchurl {
+      url = "https://github.com/raelgc/scudcloud/archive/v${version}.tar.gz";
+      sha256 = "0yzr8pc2n9305liyjw07qqja79wfrl0kp2yqk41rkcrqg6fxcyyw";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pyqt4 dbus ];
```

Comment:
No, qt5 isn't working too at my system -- crashes on connect because of some SSL verification failure.